plan/exhaust_physical_plans.go

```diff
@@ -540,36 +540,48 @@ func (p *LogicalJoin) buildRangeForIndexJoin(indexInfo *model.IndexInfo, innerPl
 		}
```

Comment:
when will `expression.Contains(accesses, eqCond)` be false

Reply:
`t1.a=t2.a and t1.c=t2.c`, and index (t1.a, t1.b, t1.c). This case currently won't use index join.